app/src/routes/transfer/(components)/transfer-form.svelte

```diff
@@ -506,8 +445,15 @@ const transfer = async () => {
 
     if ($transferState.kind === "CONFIRMING_TRANSFER") {
       try {
-        const transferHash = await writeContract(config, $transferState.contractRequest)
-        transferState.set({ kind: "AWAITING_TRANSFER_RECEIPT", transferHash })
+        const transfer = await unionClient.transferAsset({
+          autoApprove: false,
+          amount: parsedAmount,
+          recipient: $recipient,
```

Comment:
recipient is supposed to be called receiver, did we also call it recipient in the sdk?

Reply:
yes everywhere in the sdk it's recipient at the moment. Change now or subsequent pr? I agree that it should be receiver everywhere